src/Common/src/System/Security/Cryptography/Asn1Reader/AsnCharacterStringEncodings.cs

```diff
@@ -103,7 +103,7 @@ public override unsafe int GetCharCount(byte* bytes, int count)
         }
 
         public
-#if netcoreapp || uap || NETCOREAPP || netstandard21
+#if netcoreapp || netcoreapp30 || uap || NETCOREAPP || netstandard21
```

Comment:
How did you find these? Was it only because they would not build? If so, I think we should probably do an all-up search for al ifdefs that have netcoreapp as a constant and add netcoreapp30 as well

Reply:
> Was it only because they would not build?

yes most of them.

> If so, I think we should probably do an all-up search for al ifdefs that have netcoreapp as a constant and add netcoreapp30 as well

I did an search and added netcoreapp as a constant in files in non-test projects

Reply:
Ok that's good, make sure that all places that had an ifdef for netcoreapp have one for netcoreapp30 as well

Reply:
FWIW, this file is manually shared with a prerelease/experimental package in corefxlab.  So even if it looks like things aren't used by corefx they might be built in that mode in corefxlab.